stl/inc/chrono

```diff
@@ -5171,6 +5177,165 @@ namespace chrono {
         basic_istream<_CharT, _Traits>& _Is, const _Time_parse_iomanip<_CharT, _Traits, _Alloc, _Parsable>& _Tpi) {
         return _CHRONO from_stream(_Is, _Tpi._Fmt.c_str(), _Tpi._Tp, _Tpi._Abbrev, _Tpi._Offset);
     }
+
+#ifdef __cpp_lib_concepts
+    // [time.format]
+
+    // clang-format off
+    template <class _Ty, class _CharT>
+    concept _Chrono_parse_spec_callbacks = requires(_Ty _At, basic_string_view<_CharT> _Sv, _Align _Aln) {
+        { _At._On_align(_Aln) } -> same_as<void>;
+        { _At._On_fill(_Sv) } -> same_as<void>;
+        { _At._On_width(int{}) } -> same_as<void>;
+        { _At._On_dynamic_width(size_t{}) } -> same_as<void>;
+        { _At._On_dynamic_width(_Auto_id_tag{}) } -> same_as<void>;
+        { _At._On_precision(int{}) } -> same_as<void>;
+        { _At._On_dynamic_precision(size_t{}) } -> same_as<void>;
+        { _At._On_dynamic_precision(_Auto_id_tag{}) } -> same_as<void>;
+        { _At._On_conversion_spec(_CharT{}, _CharT{}) } -> same_as<void>;
+        { _At._On_lit_char(_CharT{}) } -> same_as<void>;

```

Comment:
these need the same treatment as _Parse_specs_callbacks w.r.t using a combination of the narrower concepts.